deploy/kiali/kiali_cr.yaml

```diff
@@ -520,9 +520,10 @@ spec:
 # config_map_name: The name of the istio control plane config map. It defaults to `istio`.
 # envoy_admin_local_port: The port which kiali will open to fetch envoy config data information.
 # istio_identity_domain: The annotation used by Istio to identify domains.
+# istio_injector_name: The name of the istio-sidecar-injector config map. This defaults to `istio-sidecar-injector`
```

Comment:
We don't add verbosity to these help strings by specifying defaults because all the values you see here ARE the defaults (we are very careful in this file to set the commented values to be the actual defaults as explained [at the top of this file here](https://github.com/kiali/kiali-operator/blob/master/deploy/kiali/kiali_cr.yaml#L16-L19)). So you will see down below you have `istio_injector_name: "istio-sidecar-injector"` - that value IS the real default. No need to call it out again here, just makes more noise in the docs. Also, shouldn't the name be more descriptive? If this is a literal config map name, shouldn't this be called "istio_injector_config_map_name" ?? I dunno, just weird to me if its a config map name it should be named that.

```suggestion
# istio_injector_name: The name of the istio-sidecar-injector config map.
```